src/map/status.h

```diff
@@ -1923,6 +1923,7 @@ enum scb_flag
 	SCB_RANGE   = 0x10000000,
 	SCB_REGEN   = 0x20000000,
 	SCB_DYE     = 0x40000000, // force cloth-dye change to 0 to avoid client crashes.
+	SCB_BODY    = 0x80000000, // Force bodysStyle change to 0
```

Comment:
Actually, this was not needed currently, since no SC does this, so maybe its best to leave that commented with proper explanation?